fractions/microprofile/microprofile-health/src/main/java/org/wildfly/swarm/microprofile/health/api/Monitor.java

```diff
@@ -54,6 +54,12 @@ static Monitor lookup() throws NamingException {
 
     Object getHealthReporter();
 
+    void unregisterContextClassLoader();
+
+    void registerContextClassLoader(ClassLoader classLoader);
+
+    ClassLoader getContextClassLoader();
+
```

Comment:
Just wondering if we want to make this part of the public API?

Reply:
That should be fine, all that class is there to facilitate the interaction between various microprofile health parts as far as I understand, not really meant for the public use. May be it should be noted n the docs somehow

Reply:
I agree it's not _meant_ for public use, but IIUC, it _is_ exposed publicly.

Reply:
Well, all public classes/methods/etc. are exposed and can be considered a part of the API ;-)

Reply:
I don't mind either way :-). I recall looking at that class and I'm not any of its methods could be meaningfully used by the non Thorntail code...

Reply:
We actually have a delineation between public API and private details, e.g. the `runtime` packages are not considered public API. The delineation is defined by the script that generates Javadoc :-)

Reply:
But @sberyozkin is right, this API isn't really all that useful and is "here be dragons" already :-)